analyzer/src/funTest/assets/projects/synthetic/dotnet-expected-output.yml

```diff
@@ -147,6 +147,7 @@ packages:
     path: ""
 issues:
 - timestamp: "1970-01-01T00:00:00Z"
-  source: "nuget-API"
-  message: "foobar:1.2.3 can not be found on Nugets RestAPI."
+  source: "NuGet"
```

Comment:
Hmm, aren't these existing hints good enough?

```
    - Use URL types that support GZIP compression.
    - Fix package names to always be lower-case to avoid unnecessary package
      searches.
    - Get independent URLs in parallel using coroutines.
```

Reply:
To me it is not obvious at all what impact only the GIP compression or only the downloading in parallel would have.

Reply:
Compressed data = smaller data = less data to transfer from the server = faster download time. And downloading two files in parallel is faster than downloading them sequentially.

Reply:
Hm, you're not really assuming I'm that stupid I believe, so it has to be a misunderstanding.
But I don't think it's worth spending the effort clarifying it, so if you're fine let's just keep this aspect of the commit message `as-is`.

Reply:
Hmm, maybe the missing bit here is that NuGet requires really *a lot* of HTTP requests to be made to the registry?